line_extractor:
    canny_high_threshold: 35000.0
    canny_threshold_ratio: 2.5
    canny_sobel_size: 7
    hough_rho_resolution: 1.0
    hough_theta_resolution: 0.0087 # 1 degree
    hough_thresh_fraction: 0.3

grid_line_estimator:
    debug_edges: true
    debug_lines: true
